content/blog/2022-07-01-Galaxy-Wellbeing-Blog-2/index.md

```diff
@@ -0,0 +1,31 @@
+date: "2022-07-01"
+---
+
+In our last blog post, we wrote a brief introduction to what the [Galaxy Well-being and Mental Health project](https://galaxyproject.org/blog/2022-06-29-galaxy-wellbeing-project/) is and what it entails. Now let’s break down the key terms we are using in this project.
+
+# What is Well-being?
+
+**Well-being** can be defined as the overall sense of health characterized by positive functioning. It has two aspects which are subjective and objective well-being that crucially recognises parts of our lives that we determine ourselves through our potential, how we feel about ourselves, how we form meaningful relationships and our sense of purpose as well as aspects such as economy, health, education and so on. It is considered to constitute 8 main dimensions which are emotional, social, occupational, physical, spiritual, environmental, intellectual and financial. If any of these dimensions are neglected over time, it will be detrimental to our well-being.
+
+# What is Mental Health?
+
+**Mental health**, according to WHO, can be understood as a state of well-being in which an individual recognizes his or her abilities or potential, can cope with everyday stressors, can work effectively, and contribute to his or her community. Mental health is critical to our individual and collective abilities to think, feel, communicate with one another, and enjoy life. Mental health promotion can be viewed as a critical issue for people, communities, and societies all over the world. Mental health problems can affect anyone and can manifest in a variety of ways. Thus, fostering positive mental health benefits everyone. It is not a one-off process but a continuous work which requires consistency and patience.
+
+# What is Diversity, Equity and Inclusion (DEI)?
+
+**Diversity** comprises all the ways in which individuals or groups differ. It includes age, ability, national origin, religion, socioeconomic status, sexual orientation, language, education, marital status, physical appearance, values, outlook and such. Diversity is beneficial in so many ways as it promotes the existence of different perspectives and experiences which accelerates the growth of the individual as well as the organization and increases creativity, productivity and overall success.
+
+**Equity** is a practice that promotes fair and equitable access, opportunities, treatment, and results for everyone. It aims to create the desired outcome for everyone involved by allocating resources tailored to one’s needs. Equity recognizes that everyone is not the same and comes from diverse backgrounds with different experiences. Equity should not be confused with the concept of equality where everyone is given the same access and resources. As expressed by this quote “Equality is giving everyone the same pair of shoes. Equity is giving everyone a pair of shoes that fits.”
+
+![Equity_Image](content/blog/2022-07-01-Galaxy-Wellbeing-Blog-2/Equity_Image.png)
```

Comment:
I think that the tests are failing because of the path, see how we did it in the other post here: https://github.com/galaxyproject/galaxy-hub/pull/1502/files#diff-c2b939b3d5584e42074a7abfa591f0b1e5910512293bb431cc8ef14e7916466aR13

Reply:
Yeah I'm editing this on gitpod now.